README.md

```diff
@@ -15,29 +15,29 @@ as **T-F-D**:
 ## Types
 
  | Type code   | Referencing        |
- | ----------- | ------------------ |
- | 0           | Peer ID            |
- | 1           | Message ID         |
- | 2           | Blob ID            |
- | 3           | Diffie-Hellman key |
- | 4           | Signature          |
- | 5           | Encrypted data     |
- | 6           | Generic data       |
-
-### Peer ID formats
-
-A peer ID TFD represents the public portion of a cryptographic keypair used to
+ |:-----------:| ------------------ |
+ | 0           | feed               |
+ | 1           | msg                |
+ | 2           | blob               |
```

Comment:
Would be nice to preserve what was before, i.e. uppercase. Otherwise it feels like it's undoing what I did here https://github.com/ssb-ngi-pointer/ssb-binary-field-encodings-spec/commit/11f7d16ecb2b9f19e684463004fc984c179a318b

Reply:
I was following the principle of keeping the string the new ad the steering names in the bfe.json
So that people using NAMED_TYPES can just look at this table and type it without having to guess the case, spaces, hyphens

Reply:
> So that people using NAMED_TYPES can just look at this table and type it without having to guess the case, spaces, hyphens

There is bfe.json for that.

Reply:
Yeah I'm i at what you mean. I think i was being paranoid expecting people to get confused. But if you think it's fine I'm happy for readme to be more human descriptive

Reply:
Oh, you know what? We could have a third column that reflects the name used in bfe.json